### PR TITLE
docs(opencollective): note that the payment process has been updated

### PR DESCRIPTION
**Pull request by mikehardy**
people should re-read it, update comment to say same

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process/_compare/89803528a2ea9c4db75ae2a2dd267931c97ffeed...08ec2a85d18304b7ef3599218327c1d1ace59c20

### CODE REVIEW

**Comment by david-allison** [comment]
Is it feasible to use a GitHub formatted admonishment?

Either way, I'm happy

**Review by david-allison — APPROVED** [review]

**Comment by mikehardy** [comment]
@david-allison yes it should be? What did you have in mind? I'm open to anything if you just want to propose+commit it (then I'll squash). The comment is the equivalent of what you'd put in an edit box though, so github-flavored markdown should work so long as it doesn't scope-out of the javascript string ?

**Comment by david-allison** [comment]
I'd use 

> [!NOTE]
> TEXT

On balance, I think this is worth blocking over, as users will have banner blindness

**Review by lukstbit — APPROVED** [review]
The notice is now more visible as it should. How it looks:

notice


Maybe change _Note_ to _Important_ ?

File: .github/workflows/opencollective_notices.yml [hunk 1]
```diff
@@ -50,6 +50,9 @@ jobs:
                       If you are interested in compensation for this work, the process with details is here:
 
                       https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid
+                    
+                      >[!NOTE]                      
+                      >PLEASE NOTE: The process was updated in August 2024. You should re-read the Payment Process page.
```
Comment (mikehardy) [inline .github/workflows/opencollective_notices.yml:55]:
```suggestion
                      >[! IMPORTANT]                      
                      >PLEASE NOTE: The process was updated in August 2024. Re-read the Payment Process page if you have not already.
```

File: .github/workflows/opencollective_notices.yml [hunk 2]
```diff
@@ -50,6 +50,9 @@ jobs:
                       If you are interested in compensation for this work, the process with details is here:
 
                       https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid
+                    
+                      >[! IMPORTANT]
```
Comment (mikehardy) [inline .github/workflows/opencollective_notices.yml:54]:
```suggestion
                      > [!IMPORTANT]
```

File: .github/workflows/opencollective_notices.yml [hunk 3]
```diff
@@ -50,6 +50,9 @@ jobs:
                       If you are interested in compensation for this work, the process with details is here:
 
                       https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid
+                    
+                      > [!IMPORTANT]
+                      >PLEASE NOTE: The process was updated in August 2024. Re-read the Payment Process page if you have not already.
```
Comment (mikehardy) [inline .github/workflows/opencollective_notices.yml:55]:
```suggestion
                      > PLEASE NOTE: The process was updated in August 2024. Re-read the Payment Process page if you have not already.
```

**Review by mikehardy — COMMENTED** [review]
@lukstbit I like the idea of important vs note
I also changed the wording on the suggestion to re-read the doc so it reads well (I hope) whether you have read it or not, versus always saying you should read it

I believe I have formatting correct and will certainly squash at this point 😆

**Pull request merged**